### PR TITLE
Fixed Nginx and updated Consul image

### PR DESCRIPTION
**Pull request by misterbisson**
- replaced containerpilot.json with co-process and telemetry
- changed Nginx health check URL to more specific path to reduce likelihood of conflicts
- update Consul image to https://github.com/autopilotpattern/consul
- add build notes

If merged, it should be tagged as `4.5.3-r2`.

### CODE REVIEW

**Comment by tgross** [comment]
LGTM except for my concerns in https://github.com/autopilotpattern/nginx/pull/18#discussion_r71872292

File: docker-compose.yml [hunk 1]
```diff
@@ -78,20 +78,18 @@ nginx:
 
 # Consul is the service catalog that helps discovery between the components
 consul:
-  image: progrium/consul:latest
-  command: -server -bootstrap -ui-dir /ui
+  image: autopilotpattern/consul:latest
+  command: >
+    /usr/local/bin/containerpilot
+    /bin/consul agent -server
+    -bootstrap-expect 1
```
Comment (misterbisson) [inline docker-compose.yml:85]: @tgross this works to create a one-node Consul cluster that we can then scale to N nodes via Docker Compose.

Comment (tgross) [inline docker-compose.yml:85]: I don't think that works -- when a node comes up if it expects only 1 and the CNS name it resolves is itself, then it'll form a split brain cluter.

Comment (misterbisson) [inline docker-compose.yml:85]: Crap, you're right about that.

Comment (misterbisson) [inline docker-compose.yml:85]: Fix incoming.

**Pull request merged**